### PR TITLE
Covid Functionality

### PR DESCRIPTION
**Pull request by muhammad-ahsan099**

### CODE REVIEW

File: covidanalyzer.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import csv
+
+
+def total_country_cases_deaths(total_country,
```
Comment (ahsanshafiq742) [inline covidanalyzer.py]: calculate_recovry_deat_count

File: covidanalyzer.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                               total_cases,
+                               total_deaths,
+                               total_recovered_cases):
+    with open('covid_cases_stats.csv') as file_obj:
```
Comment (ahsanshafiq742) [inline covidanalyzer.py]: covid_data

File: covidanalyzer.py (hunk 1 above)
Comment (ahsanshafiq742) [reply covidanalyzer.py]: remove all parameters

File: covidanalyzer.py [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        reader_obj = csv.DictReader(file_obj)
+        for row in reader_obj:
+            if row['country'] != 'Total:':
+                if row['country']:
+                    total_country += 1
```
Comment (ahsanshafiq742) [inline covidanalyzer.py]: It will return the total number of rows as country count

File: covidanalyzer.py [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+                if row['total_deaths']:
+                    total_deaths += int(row['total_deaths'])
+                if row['total_cases']:
+                    total_cases += int(row['total_cases'])
+                if row['total_recovered']:
+                    total_recovered_cases += int(row['total_recovered'])
+            else:
+                pass
```
Comment (ahsanshafiq742) [inline covidanalyzer.py]: If we don't have any condition to add in else then we don't have to add it
Remove else and pass

File: covidanalyzer.py [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+        return total_country, total_cases, total_deaths, total_recovered_cases
```
Comment (ahsanshafiq742) [inline covidanalyzer.py]: return a dictionary instead of many values

File: covidanalyzer.py [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+            if str(country_name != 'Total'):
+                if str(country_name) == "-a " + row['country']:
+                    print("-a " + row['country'])
+                    getting_ratio = int(row['total_recovered']) / int(row['total_cases'])
+                    print('Recovered/total ratio: ', round(getting_ratio, 2))
+                    break
+                else:
+                    print("Country Does Not Find")
+                    break
+            else:
```
Comment (ahsanshafiq742) [inline covidanalyzer.py]: remove else and pass

File: covidanalyzer.py [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+    with open('covid_cases_stats.csv') as file_obj:
+        reader_obj = csv.DictReader(file_obj)
+        for row in reader_obj:
+            if str(country_name != 'Total'):
+                if str(country_name) == "-a " + row['country']:
+                    print("-a " + row['country'])
+                    getting_ratio = int(row['total_recovered']) / int(row['total_cases'])
+                    print('Recovered/total ratio: ', round(getting_ratio, 2))
+                    break
+                else:
```
Comment (ahsanshafiq742) [inline covidanalyzer.py]: we must not have this else

File: covidanalyzer.py [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+
+        return total_country, total_cases, total_deaths, total_recovered_cases
+
+
+def given_country_stats(country_name):
+    with open('covid_cases_stats.csv') as file_obj:
+        reader_obj = csv.DictReader(file_obj)
+        for row in reader_obj:
+            if str(country_name != 'Total'):
+                if str(country_name) == "-a " + row['country']:
```
Comment (ahsanshafiq742) [inline covidanalyzer.py]: remove -a

File: covidanalyzer.py [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+                    print("-a " + row['country'])
+                    getting_ratio = int(row['total_recovered']) / int(row['total_cases'])
+                    print('Recovered/total ratio: ', round(getting_ratio, 2))
+                    break
```
Comment (ahsanshafiq742) [inline covidanalyzer.py]: return ratio value

**Review by ahsanshafiq742 — CHANGES_REQUESTED** [review]

File: covidanalyzer.py (hunk 3 above)
Comment (muhammad-ahsan099) [reply covidanalyzer.py]: Resolved..

File: covidanalyzer.py (hunk 1 above)
Comment (muhammad-ahsan099) [reply covidanalyzer.py]: Resolved..

File: covidanalyzer.py (hunk 4 above)
Comment (muhammad-ahsan099) [reply covidanalyzer.py]: Resolved..

File: covidanalyzer.py (hunk 5 above)
Comment (muhammad-ahsan099) [reply covidanalyzer.py]: Resolved..

File: covidanalyzer.py (hunk 6 above)
Comment (muhammad-ahsan099) [reply covidanalyzer.py]: Resolved..

File: covidanalyzer.py (hunk 7 above)
Comment (muhammad-ahsan099) [reply covidanalyzer.py]: Resolved..

File: covidanalyzer.py (hunk 8 above)
Comment (muhammad-ahsan099) [reply covidanalyzer.py]: Resolved..

File: covidanalyzer.py (hunk 9 above)
Comment (muhammad-ahsan099) [reply covidanalyzer.py]: Resolved..

File: covidanalyzer.py (hunk 2 above)
Comment (muhammad-ahsan099) [reply covidanalyzer.py]: Resolved..

**Pull request closed without merging**